apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/ArrayAsListAssertionsSpec.kt

```diff
@@ -0,0 +1,84 @@
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.notImplemented
+
+class ArrayAsListAssertionsSpec : ch.tutteli.atrium.specs.integration.ArrayAsListAssertionsSpec(
+    "asList",
+    Companion::arrayInt,
+    Companion::byteArray,
+    Companion::charArray,
+    Companion::shortArray,
+    Companion::intArray,
+    Companion::longArray,
+    Companion::floatArray,
+    Companion::doubleArray,
+    Companion::booleanArray,
+
+    Companion::arrayIntWithCreator,
+    Companion::byteArrayWithCreator,
+    Companion::charArrayWithCreator,
+    Companion::shortArrayWithCreator,
+    Companion::intArrayWithCreator,
+    Companion::longArrayWithCreator,
+    Companion::floatArrayWithCreator,
+    Companion::doubleArrayWithCreator,
+    Companion::booleanArrayWithCreator
+) {
+
+    companion object {
+        fun arrayInt(plant: Expect<Array<Int>>) = plant asList o
+        fun byteArray(plant: Expect<ByteArray>) = plant asList o
+        fun charArray(plant: Expect<CharArray>) = plant asList o
+        fun shortArray(plant: Expect<ShortArray>) = plant asList o
+        fun intArray(plant: Expect<IntArray>) = plant asList o
+        fun longArray(plant: Expect<LongArray>) = plant asList o
+        fun floatArray(plant: Expect<FloatArray>) = plant asList o
+        fun doubleArray(plant: Expect<DoubleArray>) = plant asList o
+        fun booleanArray(plant: Expect<BooleanArray>) = plant asList o
+
+        fun arrayIntWithCreator(plant: Expect<Array<Int>>, assertionCreator: Expect<List<Int>>.() -> Unit) =
+            plant asList assertionCreator
```

Comment:
please change this line to (not that yours is wrong, I just like to see the syntax as it is used afterwards)
```suggestion
            plant asList { assertionCreator() }
```